pytential/linalg/proxy.py

```diff
@@ -531,8 +658,7 @@ def gather_block_neighbor_points(
     """
 
     if max_particles_in_box is None:
-        # FIXME: this is a fairly arbitrary value
-        max_particles_in_box = 32
+        max_particles_in_box = _DEFAULT_MAX_PARTICLES_IN_BOX
 
     # {{{ get only sources in indices
 
```

Comment:
Avoid recreating `TreeBuilder` and `AreaQueryBuilder` in `gather_block_neighbor_points`?

Reply:
https://github.com/inducer/pytential/pull/143